simulation/components/GarrisonHolder.js

```diff
@@ -185,13 +188,22 @@ GarrisonHolder.prototype.IsAllowedToGarrison = function(ent)
 	return MatchesClassList(entityClasses, this.template.List._string) && !!Engine.QueryInterface(ent, IID_Garrisonable);
 };
 
+/**
+* Return string of allowed classes in the form "Class1 Class2+Class3" from visibleGarrisonPoint to handle default value
+*/
+GarrisonHolder.prototype.GetVisibleAllowedClasses = function (visibleGarrisonPoint)
+{
+	if (!visibleGarrisonPoint)
+		return "";
+	return visibleGarrisonPoint.offset.allowedClasses._string || visibleGarrisonPoint.offset.allowedClasses;
```

Comment:
Why is that double check needed ? Is there a case where one of them are  null

Reply:
If I use "Infantry+Ranged" above, _string is not defined, the second part is there just for that one case